[GLOSSARY] block (#437)
zkSNACKs/WasabiDoc: docs/glossary/Glossary.md

Read more: [Wasabi Supported BIPs](/using-wasabi/BIPs.md)
:::

:::details
### Block

> A block is a batch of valid Bitcoin transactions and it must hash to a value below a certain difficulty target to proof the work of the miner.

Reviewer: A block is a batch of valid Bitcoin transactions and its hash must be a value below a certain difficulty target to proof the work of the miner.

Author: ```suggestion
A block is a batch of valid Bitcoin transactions and its hash must be a value below a certain difficulty target to proof the work of the miner.
```

> A block is a batch of valid Bitcoin transactions and its hash must be a value below a certain difficulty target to proof the work of the miner.

Reviewer 2: ```suggestion
A block is a batch of valid Bitcoin transactions and its hash must be a value below a certain difficulty target to prove the work of the miner.
```

Each block references the hash of the previous block, thus creating a hash chain of blocks.

> On average a new block is created every 10 minutes.

Reviewer 2: ```suggestion
On average a new block is mined (created) every 10 minutes.
```

Author: ```suggestion
On average a new block is mined every 10 minutes.
```

> A block can be up to 4 weight units [or 4 virtual bytes] in size.

Reviewer 2: Maybe we should remove this because the size of the block may change.

Author: ```suggestion
```
